### ISSUE TITLE
Remove workaround for: Dialog box flies off screen or disappears immediately after popping up

### ISSUE THREAD

**Issue by PeterW-gif** [Extended-Thunder/send-later#663]
### Workaround (added by maintainer to original bug report)

Try moving the compose window away from the right edge of the screen.

Try making the compose window smaller.

Customize the compose window toolbar and remove the Send Later button. You will then need to use the File > Send Later menu command or Ctrl/Command-Shift-Enter to activate Send Later, but the window that pops up will no longer be linked to the button and therefore this problem will not occur.

### Describe the bug

Since updating to TB 115 on my MacBook Air when I try to use send later to send a message I've composed, the Send Later dialog box pops-up very briefly and then appears to fly across my screen and disappears off the right hand side. Therefore I cannot click on it to complete the send later action.
I run the same on my Windows 10PC and it does NOT have this problem, on the Windows PC the dialog box pops up at the top left of the screen and remains there until I fill it in and click on Send Later - so no problem.
I am running TB115.4.1 (it was the same with 115.3) and Send Later 10.3.3 on both machines.
Before updating from TB 102 I did not have this Macbook only problem, it just started on the Macbook after updating to TB115

### How to reproduce

On a Macbook Air Compose an email
Click Send Later in the toolbar
The send later dialog box should briefly pop-up on the left of the screen but before it can be accessed, then fly across the screen and disappear off the right hand side

### Expected behavior

I expected that the dialog box would pop-up and remain on the screen so I could enter the time to send and then click send later - which is what it does Ok on my Windows 10PC. As I said this problem only happens on my Macbook Air and only since updating Thunderbird from 102 to 115

### Operating System

MacOs

### Operating System Version

Ventura 13.5.2

### Thunderbird version

115.4.1

### Send Later start-up string

Sorry I'm submitting this report from my Windows PC not the Macbook which has the problem, so am unable to send you this

**Comment by bzark** [comment]
I see this on Sonoma 14.1 as well.  It has something to do with the "new/reply-to message" being full maximized.  If that "new message" window is not fully maximized then it works properly.

**Comment by PeterW-gif** [comment]
Yes after my original post I upgraded to Sonoma 14.1 but the problem persisted. Having read Bzark's post I 'umaximized' my write window (found I had to change the maximise green button to 'zoom' using the opt key). Then as bzark said it worked properly. However this is still just a Mac issue as I can use a maximised window on Windows 10 with no problem. Also the problem only started after the upgrade from TB102 toTB115 - I'm pretty sure I was using a maximised write window on 102 with no problem as with the smaller screen size of my Macbook I usually have the windows maximised. Anyway a good workaround - so thanks to bzark

**Comment by luiseduardobraschi** [comment]
> However this is still just a Mac issue

I'm having a similar issue on Ubuntu 20.04.

**Comment by oleole39** [comment]
Not fully sure whether it is related, but on Linux Mint 21 (Ubuntu 22.04 LTS based) with TB 102.15.1 (64bits) and having recently updated to Send Later 10.3.3, the dialog box (which in my case would previously appear on the top right corner below the Send Later button) appears centered in the screen. I find it more convenient as it was before.

**Comment by jikamens** [comment]
>Not fully sure whether it is related, but on Linux Mint 21 (Ubuntu 22.04 LTS based) with TB 102.15.1 (64bits) and having recently updated to Send Later 10.3.3, the dialog box (which in my case would previously appear on the top right corner below the Send Later button) appears centered in the screen. I find it more convenient as it was before.
Different issue. Necessary because of a deficiency in TB102 that is fixed in TB115.

**Comment by jikamens** [comment]
Reopening so I can switch back to the modal when the TB team fixes the issue.

**Comment by jikamens** [comment]
Ref: https://bugzilla.mozilla.org/show_bug.cgi?id=1881966

**Comment by jikamens** [comment]
I think this is is probably fixed at this point.
